utils/make_readme_table.py

```diff
@@ -77,10 +77,13 @@ def display_name(self) -> str:
     ReleaseInfo("release_9", "1.5.0", "0.21.1", "November 4, 2020"),
     ReleaseInfo("release_10", "1.6.0", "0.22.0", "November 18, 2020"),
     ReleaseInfo("release_11", "1.7.0", "0.23.0", "December 21, 2020"),
+    ReleaseInfo("release_12", "1.7.1", "0.23.0", "December 22, 2020"),
 ]
 
 MAX_DAYS = 150  # do not print releases older than this many days
-sorted_versions = sorted(versions, key=lambda x: x.loose_version, reverse=True)
+sorted_versions = sorted(
```

Comment:
This wasn't ordering 11 and 12 correctly, since they had the same python version